qiskit/ignis/verification/accreditation/qotp.py

```diff
@@ -175,11 +175,11 @@ def QOTP_fromlayers(layers, rng):
         # add to circuit and reset temp
         temp_circ = transpile(temp_circ,
```

Comment:
Note that these basis gates are deprecated. Consider using instead:
basis_gates = ['cz', 'sx', 'x', 'cx']